bin/git-forgit

```diff
@@ -137,10 +137,10 @@ _forgit_diff() {
     #   oldfile\0
     # We have to do a two-step sed -> tr pipe because OSX's sed implementation does
     # not support the null-character directly.
-    get_files="echo {} | sed 's/\\\\s*\\\\[.]\\\\s*//' | sed 's/  ->  /\\\n/' | tr '\\\n' '\\\0'"
+    get_files="echo {} | sed -e 's/^[[:space:]]*\\\\[[A-Z]\\\\][[:space:]]*//' | sed 's/  ->  /\\\n/' | tr '\\\n' '\\\0'"
```

Comment:
For this, I changed it to "Remove everything that looks like a capital letter in brackets and a bunch of spaces following it"

Please sanity check if you agree that this makes sense.

I was seeing the pattern

`[M]      somefile.txt`
`[M]      another[file].txt`

Reply:
This does the trick for me:

`get_files="echo {} | sed -e 's/^[[:space:]]*\\\\[[A-Z0-9]*\\\\][[:space:]]*//' | sed 's/  ->  /\\\n/' | tr '\\\n' '\\\0'"`

The previous `[A-Z]` did not match the `R100` for renamed files.